draft-ietf-jose-hpke-encrypt.md

```diff
@@ -177,27 +178,33 @@ To avoid confusion between JWE AAD and HPKE AAD, this document uses the term "HP
 
 ## Encapsulated Keys
 
-Encapsulated keys MUST be the base64url encoded encapsulated key as defined in Section 5.1.1 of {{RFC9180}}.
+HPKE encapsulated key is defined in Section 5.1.1 of {{RFC9180}}.
 
-In HPKE JWE Integrated Encryption, JWE Encrypted Key is the encapsulated key.
+In HPKE JWE Integrated Encryption, the JWE Encrypted Key of the sole recipient is the HPKE encapsulated key.
```

Comment:
JWE implicitly does base64url on JWE Encrypted Key, so old version apparently specified double-base64url, which surely is not what is intended.